### PR TITLE
Options for amino/nuc and to write fastest sites out

### PR DESCRIPTION
**Pull request by aguang**
Options for amino/nuc and to write fastest sites out, #107

### CODE REVIEW

**Comment by robert-ervin-jones** [comment]
Hi @aguang,

Thank you so much for your contribution. It looks like you are adding a new file named `fast_site_remover.py` to the repo's root directory rather than making changes to `fast_site_remover.py` in the `utilities` directory. Can you double-check this? If this is the case, can you fix the PR so that the original `fast_site_remover.py` is being updated rather than adding a new file? 

Thanks again! If you have any questions or concerns, please don't hesitate to reach out.

Best,
Robert

**Comment by aguang** [comment]
Ah ok that's why it didn't appear to change the file (I tried to do it through uploading a file on the web interface). It should be fixed now, thank you!

File: phylofisher/utilities/fast_site_remover.py [hunk 1]
```diff
@@ -54,8 +55,18 @@ def fake_tree(treefile, pseudonames):
             original = original.replace(key, value)
         res.write(original)
 
+def control_file_nuc():
+    ctl = """treefile = TEMP.tre * treefile
+    seqfile = TEMP.phy * sequence data
+    
+    nchar = 4           * nucleic data
+    model = 2           * F81 (don't want to give rates for HKY/GTR...)
```
Comment (robert-ervin-jones) [inline phylofisher/utilities/fast_site_remover.py:63]: Can you change this to 7 (GTR)?

**Review by robert-ervin-jones — CHANGES_REQUESTED** [review]
Can we change the model to GTR? Otherwise everything else looks good.

**Comment by robert-ervin-jones** [comment]
Hi @aguang 

I apologize for just now getting back around to this.

Best,
Robert

**Review by robert-ervin-jones — APPROVED** [review]

**Pull request merged**